src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that needs to be done before a specific date/time.
+ * Includes a String representation of the date/time. Time here is optional.
+ *
+ * @author Abdulelah Faisal S Al Ghrairy
+ */
+public class Deadline extends Task {
+
+    protected LocalDate date;
+    protected LocalTime time;
```

Comment:
I think it is better for these fields to be `private` and provide corresponding getter methods.